src/screens/SettingsScreen.tsx

```diff
@@ -0,0 +1,288 @@
+                    </View>
+                    <View style={styles.buttonsThemeContainer}>
+                        <View
+                            style={[
+                                styles.buttonsTheme,
+                                { borderColor: !systemTheme && blackTheme ? colors.LIGHT_ORANGE : colors.WHITE },
+                            ]}
+                        >
+                            <TouchableOpacity
+                                onPress={() => {
+                                    changeTheme(false, true);
+                                }}
+                            >
+                                <Image style={styles.iconMoon} source={require('@assets/icons/icon-moon.png')} />
+                                <Text style={styles.textTheme}>{SettingScreenTheme.BLACK}</Text>
+                            </TouchableOpacity>
+                        </View>
+                        <View
+                            style={[
+                                styles.buttonsTheme,
+                                { borderColor: !systemTheme && !blackTheme ? colors.LIGHT_ORANGE : colors.WHITE },
+                            ]}
+                        >
+                            <TouchableOpacity
+                                onPress={() => {
+                                    changeTheme(false, false);
+                                }}
+                            >
+                                <Image style={styles.iconSun} source={require('@assets/icons/icon-sun.png')} />
+                                <Text style={styles.textTheme}>{SettingScreenTheme.LIGHT}</Text>
+                            </TouchableOpacity>
+                        </View>
+                    </View>
+                </View>
+            </View>
+        </View>
+    );
+};
+
+const styles = StyleSheet.create({
+    wrapper: {
+        backgroundColor: colors.EXTRA_LIGHT_GRAY,
+        width: '100%',
+        height: '100%',
+    },
+    headScreen: {
+        width: '100%',
+        marginTop: 40,
+        marginBottom: 44,
+        flexDirection: 'row',
+        alignItems: 'center',
+        columnGap: 80,
+    },
+    head__title: {
+        fontFamily: 'ExpandedBold',
+        fontSize: 20,
+    },
+    iconBack: {
+        marginLeft: MAIN_HORIZONTAL_OFFSET,
+    },
+    contentText: {
+        marginLeft: 20,
+        fontFamily: 'ExtendedSemiBold',
+        fontSize: 15,
+    },
+    degrees: {
+        height: 40,
+        backgroundColor: colors.WHITE,
+        borderTopLeftRadius: 12,
+        borderTopRightRadius: 12,
+        alignItems: 'center',
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+        marginBottom: 0.6,
+    },
+    cities: {
+        height: 40,
+        backgroundColor: colors.WHITE,
+        borderBottomLeftRadius: 12,
+        borderBottomRightRadius: 12,
+        alignItems: 'center',
+        flexDirection: 'row',
+        columnGap: 235,
```

Comment:
Use columnGap only when you need to separate elements by some value, not when you need to place one element at the start and another at the end, we want that to work on different device screens